libafl_qemu/libafl_qemu_sys/build_linux.rs

```diff
@@ -14,24 +14,27 @@ macro_rules! assert_unique_feature {
     }
 }
 
+#[macro_export]
+macro_rules! assert_at_least_one_feature {
+    ($($feature:literal),+) => {
+        #[cfg(not(any($(feature = $feature),+)))]
+        compile_error!(concat!("At least one of the following features must be enabled:", $(" ", $feature),+));
+    };
+}
+
 pub fn build() {
-    // Make sure that exactly one qemu mode is set
+    // Make sure that at most one qemu mode is set
     assert_unique_feature!("usermode", "systemmode");
+    // Make sure that at least one qemu mode is set
+    assert_at_least_one_feature!("usermode", "systemmode");
```

Comment:
@rmalmain these two assert should make the code down actually unreachable and print a proper message, do you prefer a `panic` anyway?

Reply:
ok we can keep it that way, it's fine

Reply:
maybe write the reason for the unreachable assumption in the unreachable macro (I think that's possible?)

Reply:
@domenukk Cool, I didn't know :+1: done